minotaur.pro

```diff
@@ -9,7 +9,7 @@ QT       += core gui
 greaterThan(QT_MAJOR_VERSION, 4): QT += widgets
 
 # Use C++11 compiler
-QMAKE_CXXFLAGS += -std=c++11
+#QMAKE_CXXFLAGS += -std=c++11

```

Comment:
this was giving me a warning ("unknown parameter" or something), and I basically read online that instead of
QMAKE_CXXFLAGS += -std=c++11

one should use

CONFIG+= c++11

I've uncommented this for now